tests/test_tile_on_grid.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+from parameterized import parameterized
+
+from monai.apps.pathology.transforms import TileOnGrid
+
+TEST_CASES = []
+# TEST_CASES.append([{'tile_count': 16, 'tile_size': 8, 'tile_filter_mode': None, 'tile_random_offset': True, 'random_seed': 123}])
```

Comment:
Please remove the test code.

Thanks.

Reply:
done

Reply:
I think the debug tests are still here?

Thanks.